jsapp/js/account/plan.component.tsx

```diff
@@ -0,0 +1,442 @@
+                          <div className={styles.iconContainer}>
+                            <Icon
+                              name='close'
+                              size='m'
+                              classNames={[styles.redClose]}
+                            />
+                          </div>
+                          {listItem.item}
+                        </li>
+                      )
+                    )}
+                  </ul>
+                )}
+                {getListItem('addons', price.name).length > 0 && (
+                  <ul>
+                    <li className={styles.listTitle}>
+                      {price.metadata.feature_addons_title}
+                    </li>
+                    {getListItem('addons', price.name).map((listItem) =>
+                      listItem.icon ? (
+                        listItem.icon === true && (
+                          <li key={listItem.item}>
+                            <div className={styles.iconContainer}>
+                              <Icon
+                                name='check'
+                                size='m'
+                                classNames={
+                                  price.name === 'Professional plan'
+                                    ? [styles.tealCheck]
+                                    : [styles.stormCheck]
+                                }
+                              />
+                            </div>
+                            {listItem.item}
+                          </li>
+                        )
+                      ) : (
+                        <li key={listItem.item}>
+                          <div className={styles.iconContainer}>
+                            <Icon
+                              name='close'
+                              size='m'
+                              classNames={[styles.redClose]}
+                            />
+                          </div>
+                          {listItem.item}
+                        </li>
+                      )
+                    )}
+                  </ul>
+                )}
+              </div>
+            )}
+          </div>
+        ))}
+
+        <div className={styles.enterprisePlan}>
+          <h1 className={styles.enterpriseTitle}> {t('Need More?')}</h1>
+          <p className={styles.enterpriseDetails}>
+            {t(
+              'We offer add-on options to increase your limits or the capacity of certain features for a period of time. Scroll down to learn more and purchase add-ons.'
+            )}
+          </p>
+          <p className={styles.enterpriseDetails}>
+            {t(
+              'If your organization has larger or more specific needs, contact our team to learn about our enterprise options.'
+            )}
+          </p>
+          <div className={styles.enterpriseLink}>
+            <a href='https://www.kobotoolbox.org/contact/' target='_blanks'>
```

Comment:
@jnm do you think this contact url should come from Back-end?

Reply:
I was talking to Igancio about the option of this also filling data to the form when it redirects as well. Specifically adding a topic  and having it fill in 'Enterprise' or whatever it may be